samples/hosts/stale_sensors.py

```diff
@@ -239,4 +238,4 @@ def hide_hosts(id_list: list) -> dict:
         for deleted in remove_result:
             print(f"Removed host {deleted['id']}")
 else:
-    print("No stale hosts identified for the range specified.")
+    print("No stale hosts identified for the range specified.")
```

Comment:
Could you add a blank line to the end of this file? (Linting) Everything else looks fine.

Reply:
Done